examples/vnet/kubernetes-master-vmss.json

```diff
@@ -0,0 +1,108 @@
+{
+    "apiVersion": "vlabs",
```

Comment:
Is this apimodel identical to `examples/kubernetes-vmss-master/customvnet.json`?

Reply:
no, it includes the following diff:
```
- "firstConsecutiveStaticIP": "10.239.255.239"
+ "agentVnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
+ "availabilityProfile": "VirtualMachineScaleSets"
```